src/pages/stacks-blockchain/testnet.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Testnet
+description: Learn about testnet phases, timelines, and features
+images:
+  large: /images/pages/testnet.svg
+  sm: /images/pages/testnet-sm.svg
+---
+
+## Introduction
+
+The Stacks 2.0 testnet is an alterative blockchain, to be used for testing. It is intended for developer experimentation and not built for production-ready integrations. The testnet is frequently reset and Stacks tokens, used as fees for operations on the testnet, can be obtained for free using a [faucet](https://www.blockstack.org/testnet/faucet).
```

Comment:
Maybe not say "alternative blockchain" but rather "Stacks 2.0 testnet is a separate (throwaway) blockchain, to be used for testing. 

Also, do we make clear that testnet Stacks are different from real Stacks.

Might be good to use "Stacks (STX)" somewhere to make that connection.

Reply:
good points, let me update!